Formula/tpm.rb

```diff
@@ -0,0 +1,25 @@
+class Tpm < Formula
+  desc "Plugin manager for tmux"
+  homepage "https://github.com/tmux-plugins/tpm"
+  url "https://github.com/tmux-plugins/tpm/archive/refs/tags/v3.1.0.tar.gz"
+  sha256 "2411fc416c4475d297f61078d0a03afb3a1f5322fff26a13fdb4f20d7e975570"
+  license "MIT"
+
+  depends_on "tmux"
+
+  def install
+    pkgshare.install Dir["*"]
+  end
+
+  def caveats
+    <<~EOS
+      To initialize TPM add this to your tmux configuration file
+      (~/.tmux.conf or $XDG_CONFIG_HOME/tmux/tmux.conf):
+        run '/opt/homebrew/share/tpm/tpm'
+    EOS
+  end
+
+  test do
+    assert_match "", shell_output("#{pkgshare}/tpm")
```

Comment:
Can you check for some actual output?

Reply:
Actual output is nothing x) I check if it run without error.